components/Modal/src/ModalDialog.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export interface ModalDialogProps {
+  children?: React.ReactNode;
+}
+
+export const ModalDialog = ({ children }: ModalDialogProps) => <div className="denhaag-modal__dialog">{children}</div>;
```

Comment:
Is er een reden waarom je niet het HTML dialog element gebruikt? (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog)